Developer_README.md

```diff
@@ -0,0 +1,210 @@
+| ------------- |:-------------:| -----:|
+| Flask      | 2.2.5 | BSD 3-Clause |
+| python_dotenv      | 0.21.1      |   BSD License (BSD-3-Clause) |
+| flask_jwt_extended      | 4.5.3      |   MIT License |
+| bcrypt      | 4.0.1      |   Apache License 2.0 |
+| flask_pymongo      | 2.3.0    |   BSD License (BSD) |
+| mongomock      | 4.1.2      |   BSD |
+| flasgger      | 0.9.7.1      |   MIT License |
+| coverage      | 7.2.7      |   Apache Software License (Apache-2.0) |
+
+  
+# What's new?
+
+This version enhanced and built on the previous version, making it 10x more interesting and easier to use!
+Here's what we added in this release.
+
+1. Ability to add custom foods and track their calories.
+2. Fully customizable meal creation.
+3. An intuitive and searchable dropdown.
+4. BMI Calculation for your personal profile.
+5. Intelligent automatic calorie burn calculation.
+6. Enhanced UI.
+7. Randomized Exercise of the Day.
+8. Google Sign-in is now an option!
+9. Better progress tracking on the landing page.
+
+## Bug Fixes
+We fixed many bugs, but here are the most notable ones...
+
+1. Sign up / Sign in issues, especially when an existing user tries to sign up again.
+2. The proxy in the config files were wrong, making it difficult to run the website. Proxy issues were fixed.
+3. Testcases were updated and fixed.
+4. Event were not un-enrollable. Now events can be enrolled and unenrolled.
+5. Exercise of the day was static, which defeats the purpose. Fixed, it is now random.
+6. Linechart on the landing page was incorrect. Fixed.
+
+:bulb: **Tip:** More enhancements/fixes can be found here : https://github.com/adi-kiran/burnout-healthgoalsapp/issues
+   
+   # Contributors
+   
+<center>
+  <table>
+    <tr>
+        <td align="center"><a href="https://github.com/sharonjoji99"><img src="https://avatars.githubusercontent.com/u/40449660?v=4" width="100px;" alt=""/><br /><sub><b>Nimal Sharon Joji</b></sub></a><br /></td>
+      <td align="center"><a href="https://github.com/adi-kiran"><img src="https://avatars.githubusercontent.com/u/36990588?v=4" width="100px;" alt=""/><br /><sub><b>Aditya Kiran</b></sub></a></td>
+    <td align="center"><a href="https://github.com/amritav"><img src="https://avatars.githubusercontent.com/u/19224584?v=4" width="100px;" alt=""/><br /><sub><b>Amrita Visalam</b></sub></a><br /></td>
+      <td align="center"><a href="https://github.com/Diksha-Maurya"><img src="https://avatars.githubusercontent.com/u/27075012?v=4" width="100px;" alt=""/><br /><sub><b>Diksha Maurya</b></sub></a><br /></td>
+    </tr>
+  </table>
+</center>
+
+  # Contribution
+  
+  Please refer the [CONTRIBUTING.md](https://github.com/adi-kiran/burnout-healthgoalsapp/blob/main/CONTRIBUTING.md) file for instructions on how to contribute to our repository.
+
+  # License
+  
+  This project is licensed under the MIT License. Please refer to [LICENSE](https://github.com/adi-kiran/burnout-healthgoalsapp/blob/main/LICENSE) for more details.
+  
```

Comment:
Is this docu the same as the readme.md?
If not, what is new/different?

Reply:
It's a more developer oriented documentation, as the name suggests. User need not know the details of the tech stack, but developers would appreciate it. Info about test cases, possible bugs, and other relevant info goes here.

Reply:
Thanks for the clarification.